src/main/java/bio/terra/janitor/service/cleanup/FlightSubmissionFactoryImpl.java

```diff
@@ -17,6 +17,7 @@ public FlightSubmission createSubmission(TrackedResource trackedResource) {
     FlightMap flightMap = new FlightMap();
     flightMap.put(FlightMapKeys.TRACKED_RESOURCE_ID, trackedResource.trackedResourceId());
     flightMap.put(FlightMapKeys.CLOUD_RESOURCE_UID, trackedResource.cloudResourceUid());
+    flightMap.put(FlightMapKeys.MDC_KEY, trackedResource.cloudResourceUid());
```

Comment:
I think it's a little funny to distinguish between the `CLOUD_RESOURCE_UID` and the `MDC_KEY` at this stage. When do we expect them to be different? Why not just look up the `CLOUD_RESOURCE_UID` in the hook and use that?

Reply:
Not needed any more